packages/react-components/react-progress/src/stories/Progress/ProgressAppearance.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { makeStyles, shorthands } from '@fluentui/react-components';
+import { Progress } from '@fluentui/react-progress';
+
+const intervalDelay = 100;
+const intervalIncrement = 0.01;
+
+const useStyles = makeStyles({
+  container: {
+    ...shorthands.padding('20px'),
+  },
+});
+
+export const Appearance = () => {
```

Comment:
This should probably be called `Label` (and maybe separate out the `Description` story)...

Reply:
That said, you might want to leave this as-is for now since we're considering using Field for the label instead of having it built into Progress. In which case, this story would get deleted.